app/assets/javascripts/nidacfrontend.js

```diff
@@ -10,4 +10,15 @@ if (window.history && window.history.replaceState && typeof window.history.repla
   window.history.replaceState(null, null, window.location.href);
 }
 
-
+// automatically submit document upload on file select
+function onFileSelect() {
+  try{
+    // NF-403 - hide the form controls, display the spinner and disable the 'Continue' button before submitting
+    document.getElementById("upload-form").className = "govuk-!-display-none";
```

Comment:
For 
- consistency with the below suggestion
- avoiding issues with removing "other" classes on the form that might be required elsewhere...

```suggestion
    document.getElementById("upload-form").classList.add("govuk-!-display-none");
```

Reply:
Thanks for both suggestions.  Should work with IE10 according to https://www.w3schools.com/jsref/prop_element_classlist.asp 

I'll give it a go

Reply:
Done and tested on IE11 using BrowserStack